react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js

```diff
@@ -1,64 +1,37 @@
-import React, { Component, Fragment } from 'react';
+import React, { useEffect, Fragment } from 'react';
 import Spinner from 'react-spinkit';
+import { useSelector, useDispatch } from 'react-redux';
 
-import api from '~services/MatchesService';
+import actionsCreators from '~redux/game/actions';
```

Comment:
When importing actions from an actions file, I'd suggest naming it something that it's clear throughout the code, because if you have many `actionCreators` you would have to name them all the same. My suggestion is:
```suggestion
import gameActions from '~redux/game/actions';
```
Which summed up would be something like: `redux/thing/actions => thingActions` or `redux/anotherThing/actions => anotherThingActions`